grails-app/conf/RestApiUrlMappings.groovy

```diff
@@ -26,19 +26,20 @@
 class RestApiUrlMappings {
 
     // grails url-mappings-report can come handy here...
+    static String version = 'v1'
 
     static mappings = {
-        '/studies'(controller: 'study', method: 'GET', resources: 'study', includes: ['index', 'show'])
+        "/$version/studies"(controller: 'study', method: 'GET', resources: 'study', includes: ['index', 'show'])
```

Comment:
Another way to factor this part out is to do something like

    '/v1' {
        '/studies'(...)
        ...
    }

I think that would be better in this case, then we can create a `/v2` block as well.